content/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-google-cloud-platform.md

```diff
@@ -70,6 +70,8 @@ This example has a job called `Get_OIDC_ID_token` that uses actions to request a
 
 This action exchanges a {% data variables.product.prodname_dotcom %} OIDC token for a Google Cloud access token, using [Workload Identity Federation](https://cloud.google.com/iam/docs/workload-identity-federation).
 
+{% raw %}
+
```

Comment:
```suggestion
{% raw %}
```

Reply:
The suggestion here doesn't seem to delete the blank line after the tag. Strange.
Maybe it will when you apply it.

Reply:
Thanks. There's something weird going on here. Neither the **Commit suggestion** not the **Add suggestion to batch** button is enabled 🤔